addons/portal/static/src/js/portal_wizard_user_field.js

```diff
@@ -0,0 +1,34 @@
+/** @odoo-module **/
+
+import { ListRenderer } from "@web/views/list/list_renderer";
+import { X2ManyField } from "@web/views/fields/x2many/x2many_field";
+import { registry } from "@web/core/registry";
+
+const { useSubEnv } = owl;
+
+export class PortalWizardUserListRenderer extends ListRenderer {
```

Comment:
Shouldn't this be in a view-specific file (``views/list``) (trying to guess a bit the owl conventions, there is probably a doc but hey, grepping in code is my preferred documentation :D ) ?